metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py

```diff
@@ -862,6 +891,45 @@ def _extract_columns(
             if optional_column:
                 yield optional_column
 
+    def _create_model(
+        self, schema: Schema, obj: RegisteredModelInfo
+    ) -> Optional[Model]:
+        if not obj.name or not obj.full_name:
+            self.report.num_tables_missing_name += 1
```

Comment:
Do you know if num_tables_missing_name is correct here? Isn't this self.report.num_ml_models_missing_name or something like this?

Reply:
nice ctached -- updated.